.builder/actions/aws_crt_java_test.py

```diff
@@ -10,9 +10,15 @@ def run(self, env):
         env.shell.setenv('AWS_CRT_MEMORY_TRACING', '2')
         actions = []
 
-        if os.system("mvn -B test -DredirectTestOutputToFile=true -DforkCount=0 \
-            -DrerunFailingTestsCount=5 -DskipAfterFailureCount=1 \
-            -Daws.crt.memory.tracing=2 -Daws.crt.debugnative=true"):
+        os.system("echo TESTING")
+        all_test_result = os.system("mvn -P continuous-integration -B test -DredirectTestOutputToFile=true -DreuseForks=false \
+            -DrerunFailingTestsCount=5 -Daws.crt.memory.tracing=2 -Daws.crt.debugnative=true")
+
+        env.shell.setenv('AWS_CRT_SHUTDOWN_TESTING', '1')
+        shutdown_test_result = os.system("mvn -P continuous-integration -B test -DredirectTestOutputToFile=true -DreuseForks=false \
+            -Daws.crt.memory.tracing=2 -Daws.crt.debugnative=true -Dtest=ShutdownTest")
```

Comment:
add a comment about why this is being run separately

actually ... maybe this is just too weird to be run as a literal unit test?
Would this be better as a `static void main()` function we can run as a standalone application?
Since it's a weird thing that doesn't work properly unless it's the ONLY test running?
It's probably fine for the code to still live under the `test/` directory

and then it would be easier to make this test more complex. Like the python test takes an arg of which step it bails out at. We could do the same thing here, running the app in a loop, bailing out at different steps